moveit_plugins/moveit_simple_controller_manager/src/moveit_simple_controller_manager_parameters.yaml

```diff
@@ -0,0 +1,103 @@
+moveit_simple_controller_manager:
+  controller_names: {
+    type: string_array,
+    default_value: [],
+    description: "List of controllers for MoveIt's controller manager",
+    validation: {
+      not_empty<>: []
+    }
+  }
+  __map_controller_names:
+    action_ns: {
+      type: string,
+      default_value: "",
+      description: "follow_joint_trajectory or gripper_cmd",
+      validation: {
+        not_empty<>: []
+      }
+    }
+    type: {
+      type: string,
+      default_value: "",
+      description: "FollowJointTrajectory or GripperCommand",
+      validation: {
+        not_empty<>: []
+      }
+    }
+    make_default: {
+      type: bool,
+      default_value: false,
+      description: "This param was originally named default. Can't use default with generate_param_lib as default is a C++ keyword. Marking a controller as
+       default makes MoveIt prefer this controller when multiple options are available.",
+      validation: {
+        not_empty<>: []
+      }
+    }
```

Comment:
I also had to rename this parameter. Originally, it was called `default` but default is a C++ keyword and won't work with how generate param library initializes variables to store the parameter values 
```
/home/abishalini/ws_moveit2/build/moveit_simple_controller_manager/moveit_simple_controller_manager_parameters/include/moveit_simple_controller_manager_parameters.hpp:73:18: error: expected unqualified-id before ‘default’
   73 |             bool default = false;
      |                  ^~~~~~~
```

Reply:
Do we need to update any config files in the moveit_resources repos or studio?